tools/webpack/blocks.js

```diff
@@ -65,15 +65,69 @@ module.exports = function( env = { environment: 'production', watch: false, buil
 	const config = {
 		...baseConfig( env ),
 		entry: {
-			'file/view': normalizeJoin(baseDir, `node_modules/@wordpress/block-library/build-module/file/view` ),
-			'navigation/view': normalizeJoin(baseDir, `node_modules/@wordpress/block-library/build-module/navigation/view` ),
-			'navigation/view-modal': normalizeJoin(baseDir, `node_modules/@wordpress/block-library/build-module/navigation/view-modal` ),
-			'search/view': normalizeJoin(baseDir, `node_modules/@wordpress/block-library/build-module/search/view` ),
+			// TODO before merging:
+			// - Rename to the final filenames, which should all be `view` and not `view-interactivity`.
+			// - Add the Search block file.
+			'navigation/view': normalizeJoin( baseDir, 'node_modules/@wordpress/block-library/build-module/navigation/view-interactivity' ),
+			'image/view': normalizeJoin( baseDir, 'node_modules/@wordpress/block-library/build-module/image/view-interactivity' ),
+			'query/view': normalizeJoin( baseDir, 'node_modules/@wordpress/block-library/build-module/query/view' ),
+			'file/view': normalizeJoin( baseDir, 'node_modules/@wordpress/block-library/build-module/file/view-interactivity' ),
```

Comment:
Just a note for posterity: the name of the generated files should match the name in the block's `block.json`.

Reply:
Yes. As the names are hardcoded in WordPress and we still have files named `view-interactivity.js`, my intention is to rename them all to `view.js` before the 16.7 RC so we don't need to modify the WordPress Webpack file in the future.

Reply:
We renamed all the files in Gutenberg and here in https://github.com/WordPress/wordpress-develop/pull/5195/commits/c9e7bb1f770862631ae0a45a192b9279d0a754c9.